test/kubernetes_test.go

```diff
@@ -3,72 +3,169 @@
 package test
 
 import (
-	"io/ioutil"
-	"log"
 	"testing"
 	"time"
 
 	"github.com/mholt/caddy"
+	"github.com/miekg/coredns/middleware/test"
```

Comment:
Nit; could you separate these out in coredns import and the rest? So stdlib, coredns, rest?